auto-exporters/logging/src/main/java/io/opentelemetry/auto/exporters/logging/LoggingExporter.java

```diff
@@ -17,10 +17,10 @@
 package io.opentelemetry.auto.exporters.logging;
 
 import io.opentelemetry.common.AttributeValue;
+import io.opentelemetry.common.ReadableKeyValuePairs.KeyValueConsumer;
 import io.opentelemetry.sdk.trace.data.SpanData;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import java.util.Collection;
-import java.util.Map;
 
 public class LoggingExporter implements SpanExporter {
```

Comment:
Not directly relevant to this PR, but why do we have one of these here, when we have one in the main repo?

Reply:
That is a great question. Please, file a bug to us :)

Reply:
will do. I think yours has much cleaner output, but it's also not using a logging framework, but just using `System.out` so maybe they're not compatible?